### PR TITLE
Reballanced Hallucinogenic Sting to inject cbz into the target

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Hallucinogenic sting remake. Now injects CBZ and BZ metabolites into your target AND yourself. Risky to move, but only way to disable other lings and prevent them from headcrabbing. Requires 3 people to absorb, to prevent lings from self-salading and just fucking with other lings for the sake of it.

## Why It's Good For The Game

Makes ling hunting AND absorbing a viable strat. You will no longer be unable to absorb other lings because they headcrab as soon as they realize what's going on.

## Changelog
:cl:
tweak: Ling hallucinogenic sting also injects some CBZ into targets. Cannot be used right away, as it requires lings to absorb 3 other targets.
/:cl:

### CODE REVIEW

**Comment by TeomanTheGreat** [comment]
This makes no sense. Why would lings have the ability to make CBZ which is literally poisonous to themselves like bruh!

**Comment by eeSPee** [comment]
> This makes no sense. Why would lings have the ability to make CBZ which is literally poisonous to themselves like bruh!

For fairness, honestly. If I made it just substract 25 points, then a ling would be able to sting the target 3 times a second and leave it at 0 DNA and unable to retaliate with ling powers.

Now, it will alert the ling target, which is able to retaliate/prepare, but won't be completely crippled instantly.

**Comment by WhyIsCaeciliusTaken** [comment]
CBZ also does a tonne of oxy damage to lings. This makes it an absolute hardcounter because you can get a lethal dose in by spam stinging before they notice. Just make last resort cost 1 point again.

**Comment by eeSPee** [comment]
I think I'll leave it at BZ metabolites then

**Comment by PowerfulBacon** [comment]
Instead of injecting BZ metabolites, you should just increase the cost of using it, since that is the same thing except one is a number that is obvious and one is kind of annoying and hard to tell why your number is frozen.

File: code/modules/antagonists/changeling/powers/tiny_prick.dm [hunk 1]
```diff
@@ -219,8 +219,8 @@
 	desc = "We induce our victim with a mixture of synthesized BZ gas. Costs 10 chemicals. Our body will absorb a part of the chemicals."
```
Comment (PowerfulBacon) [inline code/modules/antagonists/changeling/powers/tiny_prick.dm]: Update description

Comment (PowerfulBacon) [reply code/modules/antagonists/changeling/powers/tiny_prick.dm]: And helptezy

Comment (PowerfulBacon) [reply code/modules/antagonists/changeling/powers/tiny_prick.dm]: Helptext

**Comment by zeskorion** [comment]
this is a slapdash attempt at fixing the issue with last resort, in a way that doesn't actually solve it.

**Comment by ghost** [comment]
> This makes no sense. Why would lings have the ability to make CBZ which is literally poisonous to themselves like bruh!

Snakes have venom, but they still will be fucked up by it

**Comment by Doosteroni** [comment]
> > This makes no sense. Why would lings have the ability to make CBZ which is literally poisonous to themselves like bruh!
> 
> Snakes have venom, but they still will be fucked up by it

**Pull request closed without merging**